client/modules/workspace/src/JobsDetailModal/JobsDetailModal.tsx

```diff
@@ -98,6 +98,8 @@ export const JobsDetailModalBody: React.FC<{
     'Node Count':
       'nodeCount' in jobDisplay ? String(jobDisplay.nodeCount) : undefined,
     Allocation: 'allocation' in jobDisplay ? jobDisplay.allocation : undefined,
+    Reservation:
+      'reservation' in jobDisplay ? jobDisplay.reservation.arg : undefined,
```

Comment:
Let's make the inclusion of this field dynamic depending on if there's a reservation param

Reply:
From what I can tell, that's currently how it works.

Reply:
Yep good catch!